packages/components/src/LottieView/index.tsx

```diff
@@ -0,0 +1,84 @@
+import type { LegacyRef } from 'react';
+import { forwardRef, useEffect, useImperativeHandle, useRef } from 'react';
+
+import AnimatedLottieView from 'lottie-react-native';
+import { AppState } from 'react-native';
+
+import type { LottieComponentProps as LottieWebProps } from 'lottie-react';
+import type { AnimatedLottieViewProps as LottieNativeProps } from 'lottie-react-native';
+import type { AppStateStatus } from 'react-native';
+
+// Stick props the same as LottieNative by now
+type LottieViewProps = Omit<LottieWebProps, 'animationData'> & {
+  source: LottieWebProps['animationData'] | LottieNativeProps['source'];
+  style?: LottieNativeProps['style'];
+  autoPlay?: boolean;
+  resizeMode?: 'cover' | 'contain' | 'center';
+};
+
+const LottieView = forwardRef<typeof AnimatedLottieView, LottieViewProps>(
+  ({ source, autoPlay, loop, resizeMode, ...props }, ref) => {
+    const animationRef = useRef<AnimatedLottieView | null>();
+
+    const appStateRef = useRef(AppState.currentState);
+    useEffect(() => {
+      // fix animation is stopped after entered background state on iOS
+      // https://github.com/lottie-react-native/lottie-react-native/issues/412
+      const handleStateChange = (nextAppState: AppStateStatus) => {
+        if (
+          appStateRef.current &&
+          /inactive|background/.exec(appStateRef.current) &&
+          nextAppState === 'active'
+        ) {
+          animationRef.current?.play?.();
+        }
+        appStateRef.current = nextAppState;
+      };
+      const subscription = AppState.addEventListener(
+        'change',
+        handleStateChange,
+      );
+      return () => {
+        subscription.remove();
+      };
+    }, []);
+
+    useEffect(() => {
+      // animation won't work in navigate(), needs delay here
+      setTimeout(() => {
```

Comment:
could you please provide an example of using setTimeout for delayed loading?

Reply:
@huhuanming 

#### Where did this piece of code originally come from:
- [Diff](https://github.com/OneKeyHQ/app-monorepo/pull/624/commits/6ff731bb1af274827952c98931243f20c43309a9)

#### How to reproduce on iOS:
- You need to place the `LottieView` on the first page of the `TabNavigator`, there is an example:
[Example Link](https://github.com/hellohublot/app-monorepo/tree/lottieViewIOSAutoPlay)

#### Solution:
- In the old version, `autoPlay` was implemented by calling the `play` method in `componentDidUpdate`. Upgrading the `lottie-react-native` version to 6.3.0 can resolve this issue because `autoPlay` has now been moved to the native side.
https://github.com/lottie-react-native/lottie-react-native/pull/986

#### Known issues:
- We can't only pass  `width` to `LottieView`, and we must provide both `width` and `height` now.

Finally, I have already upgraded `lottie-react-native` to the latest version in this PR.